howso/utilities/feature_attributes/tests/conftest.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+from unittest.mock import patch
+
+import mongomock
+import pandas as pd
+import pytest
+
+try:
+    from howso.connectors.abstract_data import (
+        convert_data,
+        make_data_source,
+        SQLTableData,
+        MongoDBData,
+        DaskDataFrameData,
+        DataFrameData,
+        ParquetDataFile,
+        ParquetDataset,
+        TabularFile,
+    )
+    from howso.connectors.tests.utils import TemporaryDirectoryIgnoreErrors
+except (ModuleNotFoundError, ImportError):
+    pytest.skip("howso-engine-connectors not installed", allow_module_level=True)
+
+
+def mongodb_data(df) -> MongoDBData:
```

Comment:
If we're `yield`ing, then the return should be `Iterator[MongoDBData]` or `Generator[MongoDBData, None, None]`